azurerm/internal/services/machinelearning/machine_learning_inference_cluster_resource_test.go

```diff
@@ -138,10 +174,17 @@ resource "azurerm_machine_learning_inference_cluster" "import" {
 
   tags = azurerm_machine_learning_inference_cluster.test.tags
 }
-`, template)
+`, r.basic(data))
+}
+
+func (r InferenceClusterResource) templateFastProd(data acceptance.TestData) string {
+	return r.template(data, "SStandard_D3_v2", 3)
```

Comment:
@aristosvo is this a spelling mistake "SStandard_D3_v2". Should it not be: "Standard_D3_v2" - I guess you did not run this test:)

Reply:
Got me 😅 Running now!